pkg/apis/apps/v1/defaults_test.go

```diff
@@ -597,7 +597,6 @@ func TestSetDefaultStatefulSet(t *testing.T) {
 	for _, test := range tests {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
-			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.StatefulSetAutoDeletePVC, test.enablePVCDeletionPolicy)
```

Comment:
I'd probably leave that variable `enablePVCDeletionPolicy` as it was before, then you'd have here this:
```golang
if !test.enablePVCDeletionPolicy {
    featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.31"))
}
featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.StatefulSetAutoDeletePVC, test.enablePVCDeletionPolicy)
```
That's what we've been using all over the places, and this will also mean less changes :wink: